packages/eslint-plugin-wpcalypso/package.json

```diff
@@ -26,5 +26,8 @@
 	},
 	"devDependencies": {
 		"babel-eslint": "^10.1.0"
+	},
+	"scripts": {
+		"test": "jest"
```

Comment:
Shouldn't this be tested as part of `yarn test-packages` in the root? Why do we need our own `jest` call?

Reply:
I moved into the package and did `yarn test`. I expected that to work and was surprised when it didn't. With this change it does work, but if that's in violation of the repo setup I'm happy to drop the change.

Reply:
I think having a `test` command here may be a bit confusing:

- Technically we don't run this command on CI, so it can break at anytime and go unnoticed. In CI we only do `yarn test-packages` in the root.
- With the current setup, the way to run tests for a single package is `yarn test-packages packages/eslint-plugin-wpcalypso`. Doing `cd packages/eslint-plugin-wpcalypso && yarn jest` also works, but that's a trick